website/content/en/docs/building-operators/golang/advanced-topics.md

```diff
@@ -231,38 +238,7 @@ func contains(list []string, s string) bool {
 During the lifecycle of an operator it's possible that there may be more than 1 instance running at any given time e.g when rolling out an upgrade for the operator.
 In such a scenario it is necessary to avoid contention between multiple operator instances via leader election so that only one leader instance handles the reconciliation while the other instances are inactive but ready to take over when the leader steps down.
 
-There are two different leader election implementations to choose from, each with its own tradeoff.
```

Comment:
just removed the part of the sdk impl and all the rest of the texts were kept in the same way.